appveyor.yml

```diff
@@ -3,9 +3,15 @@
 # license that can be found in the LICENSE file.
 
 install:
-  - ps: .\tool\appveyor.ps1 install
+  - cinst -y dart-sdk
+  - ps: $env:PATH = "$env:PATH;c:\tools\dart-sdk\bin"
```

Comment:
If we're mostly using CMD.exe, we could here too:

    SET PATH=%PATH%;C:\tools\dart-sdk\bin